tests/test_parametric_workflow.py

```diff
@@ -125,7 +123,7 @@ def test_parametric_workflow(monkeypatch: pytest.MonkeyPatch):
     case_path = str(Path(temporary_resource_path) / "Static_Mixer_Parameters.cas.h5")
     solver_session.tui.file.write_case(case_path)
 
-    assert (Path(temporary_resource_path) / "Static_Mixer_Parameters.cas.h5").exists()
```

Comment:
@mkundu1 It's passing on windows, no idea why it is failing in Linux.

![image](https://github.com/ansys/pyfluent-parametric/assets/106588300/4afbebbd-10cc-4419-8717-1cab989a7ce6)

Reply:
After adding `@pytest.mark.self_hosted` it is passing now.